src/stylesheets/abstracts/_variables.scss

```diff
@@ -23,11 +23,16 @@ $v-globals: (
   light-brand-clr: #e4fdf9,
 
   second-brand-clr: #3949ab,
-  
 
   /// Container's minimum width
   min-width: 320px,
   max-width: 1180px,
-  min-height: 750px
+  min-height: 750px,
+
+  //Media size
+  media-xs: 576px,
```

Comment:
We already have a cool mixin for media-quires. Check out the _abstracts_ folder.
Yeah, here are the mixins:
```scss
@include respond-below(sm) {}
@include respond-above(md) {}
```
Please use them instead.